### PR TITLE
#3909: Removes articles breaking sitemap.xml

### PR DESCRIPTION
**Pull request by paulroemer**

### CODE REVIEW

File: articles/framework/demo-external.asciidoc
[the remark this answers is not in the document]
Comment (paulroemer) [reply articles/framework/demo-external.asciidoc:1]: Is this something I can do in an efficient manner to help you? If yes, can you point me on it?

[the remark this answers is not in the document]
Comment (jouni) [reply articles/framework/demo-external.asciidoc:1]: Here’s the place where we should filter out pages that have the `url` frontmatter attribute: https://github.com/vaadin/docs-app/blob/master/astro.config.mjs#L84

[the remark this answers is not in the document]
Comment (jouni) [reply articles/framework/demo-external.asciidoc:1]: @tomivirkki might know how to do that?

[the remark this answers is not in the document]
Comment (tomivirkki) [reply articles/framework/demo-external.asciidoc:1]: Something like [this](https://github.com/vaadin/docs-app/commit/5712e8cd714667d7cea723b5cb66da7c41aa755f) should do it

[the remark this answers is not in the document]
Comment (tomivirkki) [reply articles/framework/demo-external.asciidoc:1]: Opened a [PR](https://github.com/vaadin/docs-app/pull/486). @jouni ptal

[the remark this answers is not in the document]
Comment (paulroemer) [reply articles/framework/demo-external.asciidoc:1]: @jouni done!

**Review by jouni — APPROVED** [review]

**Pull request merged**